### PR TITLE
Add Morgan Stanley

### PR DESCRIPTION
**Pull request by fm**
Fixes errors with #777

### CODE REVIEW

File: companies/morganstanley.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "slug": "morganstanley",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Morgan Stanley & Co International plc",
+    "address": "Legal & Compliance Division\nMorgan Stanley & Co International plc\n20 Bank Street, Canary Wharf\nLondon E14 4AD\nUnited Kingdom",
```
Comment (baltpeter) [inline companies/morganstanley.json:10]: Please don't add the company name or ATTN line to the address, as these are added automatically by our website ([see](https://github.com/datenanfragen/data#addresses)).

File: companies/morganstanley.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "email": "dataprotectionoffice@morganstanley.com",
+    "web": "https://www.morganstanley.com",
+    "sources": [
+        "https://www.morganstanley.com/disclaimers/emea-privacy-policy"
+    ],
+    "required-elements": [
```
Comment (baltpeter) [inline companies/morganstanley.json:16]: Are you sure that all these elements are necessary? We generally try to keep these to a minimal set so as not to expose unnecessary personal data to the company when sending a request.

I have no experience with Morgan Stanley but usually the combination of name and email or name and address are sufficient.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

File: companies/morganstanley.json (hunk 1 above)
Comment (fm) [reply companies/morganstanley.json:10]: My bad, I was sure I had stripped it out.

File: companies/morganstanley.json (hunk 2 above)
Comment (fm) [reply companies/morganstanley.json:16]: You're right. The US version using CCPA requested all these details, but I've simplified it and let the request any additional data

File: companies/morganstanley.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "slug": "morganstanley",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Morgan Stanley & Co International plc",
+    "address": "Morgan Stanley & Co International plc\n20 Bank Street, Canary Wharf\nLondon E14 4AD\nUnited Kingdom",
```
Comment (baltpeter) [inline companies/morganstanley.json:10]:
```suggestion
    "address": "20 Bank Street, Canary Wharf\nLondon E14 4AD\nUnited Kingdom",
```

**Review by baltpeter — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**